scripts/policy/frameworks/storage/backend/sqlite/main.zeek

```diff
@@ -20,14 +20,28 @@ export {
 		## different between the two.
 		table_name: string;
 
-		## Key/value table for passing tuning parameters when opening the
-		## database.  These must be pairs that can be passed to the ``pragma``
-		## command in sqlite.
-		tuning_params: table[string] of string &default=table(
+		## Key/value table for passing pragma commands when opening the database.
+		## These must be pairs that can be passed to the ``pragma`` command in
+		## sqlite. The ``integrity_check`` pragma is run automatically and does
+		## not need to be included here. For pragmas without a second argument,
+		## set the value to an empty string.
+		pragma_commands: table[string] of string &ordered &default=table(
+			["integrity_check"] = "",
+			["busy_timeout"] = "5000",
 			["journal_mode"] = "WAL",
 			["synchronous"] = "normal",
 			["temp_store"] = "memory"
 		);
```

Comment:
```suggestion
		) &ordered;
```


Without,  adding an option to a default constructed `Options` record wouldn't put an additional option at the end. E.g, test with:
```
$ zeek frameworks/storage/backend/sqlite -e 'local o = Storage::Backend::SQLite::Options($database_path="x", $table_name="y"); o$pragma_commands["XXX_last_pragma"] = "XXX_last_pragma"; print o'
[database_path=x, table_name=y, pragma_commands={
[integrity_check] = ,
[XXX_last_pragma] = XXX_last_pragma,
[temp_store] = memory,
[synchronous] = normal,
[journal_mode] = WAL,
[busy_timeout] = 5000
}, pragma_timeout=500.0 msecs, pragma_wait_on_failure=5.0 msecs]
```

That's probably due to #4448 or #4189 and hopefully already on @evantypanski's list of Zeek script gotchas :-)